src/components/NotificationsDrawer/DrawerPanelContent.tsx

```diff
@@ -103,43 +101,32 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
   };
 
   const dropdownItems = [
+    <DropdownItem value={0} key="action" description="Actions"></DropdownItem>,
     <DropdownItem key="read all" onClick={onMarkAllAsRead} isDisabled={notifications.length === 0}>
       Mark visible as read
     </DropdownItem>,
     <DropdownItem key="unread all" onClick={onMarkAllAsUnread} isDisabled={notifications.length === 0}>
       Mark visible as unread
     </DropdownItem>,
     <Divider key="divider" />,
+    <DropdownItem value={0} key="action" description="Quick links">
+      ,
```

Comment:
is this comma needed?

Reply:
Yes, it throws an error without the commas . The other dropdownitems have them, as well.

Reply:
ah, I see, I think this one might be misplaced as it shows in the image above the "Quick Links" text

Reply:
good catch!